### PR TITLE
chore: update copyrights years

### PR DESCRIPTION
**Pull request by agnes512**
#### Related issues

- connect to https://github.com/strongloop-internal/scrum-apex/issues/406

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by dhmlau** [comment]
@b-admike , are you aware of this downstream failure in postgresql?  Thanks.

File: lib/connectors/transient.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Copyright IBM Corp. 2014,2018. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (dhmlau) [reply lib/connectors/transient.js]: @b-admike  FYI - the first "year" in the copyright statement is the year that this file was created, 2nd "year" is the year that the file was last updated.

[the remark this answers is not in the document]
Comment (agnes512) [reply lib/connectors/transient.js]: Yeah. But some part of it need to be fixed manually.

[the remark this answers is not in the document]
Comment (dhmlau) [reply lib/connectors/transient.js]: and yes, we're running `strong-tools` `slt copyright` command.

**Pull request merged**